server/commands.lua

```diff
@@ -31,11 +48,13 @@ lib.addCommand('addkeys', {
     },
     restricted = 'group.admin',
 }, function (source, args)
-    if not args.id or not args.plate then
-        return exports.qbx_core:Notify(source, locale("notify.fpid"))
+    local playerId, plate = getVehicleKeysParams(source, args)
+
+    if not playerId or plate == 0 then
+        return exports.qbx_core:Notify(source, locale('notify.fpid'), 'error')
     end
 
-    GiveKeys(args.id, args.plate)
+    GiveKeys(playerId, plate)
 end)
```

Comment:
I think the handling of the args is going to be more readable if instead of getVehicleKeysParams, you extract a function getPlayersVehiclePlate(src) to only return the plate of the src's vehicle, while keeping the default value args.id to src logic inside of this function.

Doing so will make it clearer how the args are being transformed/used.

Reply:
right, after making `local playerId = args.target or source` it makes even more sense